src/collectors/windows.plugin/metadata.yaml

```diff
@@ -1206,6 +1206,92 @@ modules:
               chart_type: line
               dimensions:
                 - name: uptime
+        - name: IIS W3SCV W3MP
+          description: "These metrics refer to the World Wide Web Publishing Service, the service responsible for hosting and serving web content"
+          labels:
+            - name: app
+              description: The name of the Application running.
+            - name: pid
+              description: Current PID of the process.
+          metrics:
+            - name: iis.w3scv_w3wp_active_threads
+              description: Threads actively processing requests in the worker process.
+              unit: "threads"
+              chart_type: line
+              dimensions:
+                - name: threads
+            - name: iis.w3scv_w3wp_requests_total
+              description: Maximum number of threads to which the thread pool can grow as needed.
+              unit: "requests"
+              chart_type: line
+              dimensions:
+                - name: requests
+            - name: iis.w3scv_w3wp_requests_active
+              description: Current number of requests being processed by the worker process.
+              unit: "requests"
+              chart_type: line
+              dimensions:
+                - name: requests
+            - name: iis.w3scv_w3wp_file_cache_mem_usage
+              description: Current memory usage by the worker process.
+              unit: "bytes/s"
+              chart_type: line
+              dimensions:
+                - name: used
```

Comment:
Why "bytes/s" if this is current memory?

Reply:
I missed updating the metadata for this, but the code was already correct.